src/components/serviceNavigator/ServiceNavigator.js

```diff
@@ -3,7 +3,7 @@ import ServiceCategoryList from './ServiceCategoryList'
 import ServiceCategorySectionList from './ServiceCategorySectionList'
 import ServiceCategoryService from '../../services/ServiceCategoryService'
 import serviceCategories from '../../mock/data/service_categories.mock.json'
-
+import serviceCards from './ServiceCards'
 const ServiceNavigator = () =>
     <div>
         <div className="row">
```

Comment:
Opportunistic a few lines below:

Update the href for Sign Up and Log in to be the register/login pages (since we have those now)